### PR TITLE
Modified field names in fake data template (fixes #199)

### LINKED ISSUE #199: Out-of-date field names in webservice test files
In fixing `test_response.py` (issue #165 ), I updated [this input file](https://github.com/DataBiosphere/azul/blob/develop/test/service/data/test1index.json) to match the current expected schema in `hca_response_v5.py`. In fixing `test_pagination.py` (issue #164 ), I had to update [the project section](https://github.com/DataBiosphere/azul/blob/develop/test/service/data_generator/fake_data_template.json#L38) to the new schema, but I suspect `fake_data_template.json` and other template input files for the webservice are out-of-line with regards to the current schema. 

Two courses of action:
1) Just update the templates appropriately
2) Introduce tests to check that the test inputs align with the in-use schema.

┆Issue is synchronized with this [JIRA Story](https://ucsc-cgl.atlassian.net/browse/AZUL-145)
┆Project Name: azul
┆Issue Number: AZUL-145

### PR DESCRIPTION
**Pull request by alex-hancock**

### CODE REVIEW

File: test/service/test_response.py [hunk 1]
```diff
@@ -41,64 +46,59 @@ def test_file_search_response(self):
...
-        with open('{}/filesearch_test1.json'.format(self.data_folder_filepath)) as test1:
-            file_search_test = json.load(test1)
+        with open(self.facet_input_filename) as facet_input_fp:
+            facet_input = json.load(facet_input_fp)
 
-        # This is what will be used as the comparing standard
-        with open('{}/facets_test_input1.json'.format(
-                self.data_folder_filepath)) as facet_input:
-            facet_test = json.load(facet_input)
+        with open(os.path.join(self.data_folder_filepath, "response_pagination_input1.json")) as pagination_input_fp:
```
Comment (hannes-ucsc) [inline test/service/test_response.py]: This statement is a bit long. Can you extract the argument to open() into a variable?

Comment (hannes-ucsc) [reply test/service/test_response.py]: Here and everywhere else.

Comment (alex-hancock) [reply test/service/test_response.py]: Addressed, extracted filepaths to setUpClass

File: test/service/test_response.py [hunk 2]
```diff
@@ -41,64 +51,59 @@ def test_file_search_response(self):
...
-            pagination_test = json.load(pagination_input)
+        with open(self.filesearch_output1_filename) as filesearch_output_fp:
+            filesearch_output = json.load(filesearch_output_fp)
 
         # Still need a way to test the response.
-        file_search_response = FileSearchResponse(
-            test_json,
-            pagination_test,
-            facet_test
+        filesearch_response = FileSearchResponse(
```
Comment (hannes-ucsc) [inline test/service/test_response.py]:
```
FileSearchResponse(
   hits_inputs=self.load('response_test_input.json'),
…
```

File: test/service/data_generator/fake_data_template.json [hunk 3]
```diff
@@ -27,11 +27,11 @@
             "hca_id": [
               "uuid4"
             ],
-            "instrument": "safe_color_name",
```
Comment (hannes-ucsc) [inline test/service/data_generator/fake_data_template.json]: These changes look all good to me.

**Pull request merged**